crates/tx5-core/src/file_check.rs

```diff
@@ -17,6 +17,19 @@ impl FileCheck {
     }
 }
 
+fn get_loc_dir() -> Option<std::path::PathBuf> {
+    match app_dirs2::app_root(
+        app_dirs2::AppDataType::UserCache,
+        &app_dirs2::AppInfo {
+            name: "host.holo.tx5",
+            author: "host.holo.tx5",
+        },
+    ) {
+        Ok(dir) => Some(dir),
+        _ => None,
+    }
+}
```

Comment:
Changes in this file reflect fixes for https://github.com/holochain/tx5/issues/69